doc/source/analyzing/units.rst

```diff
@@ -335,3 +335,23 @@ second example returns data in low-redshift comoving megaparsecs.
 Wherever possible it's best to do calculations in physical units when working
 with more than one dataset. If you need to use comoving units or code units then
 extra care must be taken in your code to avoid ambiguity.
+
+
+Defining custom units globally
+------------------------------
+
+Units can be added to yt at runtime using ``yt.units.define_unit``.
+Such units can then be imported from the ``yt.units`` namespace.
```

Comment:
but they won't be available in a `ds`..... which is maybe actually how it should be (see main comment), but is probably worth mentioning here.